website/docs/get-started.md

```diff
@@ -85,7 +85,7 @@ Each pipeline in WARP has accompanying continuous integration tests that run on
 
 ## Feedback
 
-WARP is always evolving! Please file any issues in GitHub or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) with suggestions, feedback, or questions. We are always excited to discuss cloud data processing, provenance and reproducibility in scientific analysis, new pipeline features, or potential collaborations. Don’t hesitate to reach out!
+WARP is always evolving! Please file any issues in GitHub or contact [the WARP Team](mailto:warp-pipelines-help@broadinstitute.org) with suggestions, feedback, or questions. We are always excited to discuss cloud data processing, provenance and reproducibility in scientific analysis, new pipeline features, or potential collaborations. Don’t hesitate to reach out!
```

Comment:
We're actually planning to move away from encouraging people to reach out to us directly, but we can make that change all at once throughout the documentation